### PR TITLE
feat(editor): Add truncate directive

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary

Introduce truncate Vue directive

## Related Linear tickets, Github issues, and Community forum posts

[PAY-1972](https://linear.app/n8n/issue/PAY-1972/add-truncate-vue-directive)

## Review / Merge checklist

- [ ] PR title and summary are descriptive.
- [ ] Tests included.
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/design-system/src/directives/n8n-truncate.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { DirectiveBinding, ObjectDirective } from 'vue';
+import { truncate } from '../utils/string';
+
+export const n8nTruncate: ObjectDirective = {
```
Comment (tomi) [inline packages/design-system/src/directives/n8n-truncate.ts:4]: What does this do? A documentation + `@example` would be nice

**Review by tomi — COMMENTED** [review]
Besides missing documentation, looks good 👍

File: packages/design-system/src/directives/n8n-truncate.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/design-system/src/directives/n8n-truncate.ts:19]: Added

**Review by tomi — APPROVED** [review]
🚀

**Review by tomi — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.60.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.60.0)

**Pull request merged**